bin/distmap

```diff
@@ -455,272 +455,351 @@ sub check_inputs {
+=item B<--fastq-type> I<TYPE>
 
-default=illumina
+This option does nothing. Quality encoding is directly detected by ReadTools.
 
-=item B<--discard-internal-N>
+=item B<--disable-zipped-output>
 
-flag, if set reads having internal Ns will be discarded; default=off
+This option does nothing.
 
-=item B<--min-length>
+=back
 
-DEPRECATED: The minimum length of the read after trimming; default=40
-This argument will be removed in favor of --trim-args.
 
 
-=item B<--no-trim-quality>
+=head1 EXAMPLES
 
-DEPRECATED: toggle switch: switch of trimming of quality.
-This argument will be removed in favor of --trim-args.
+=over 6
 
-=item B<--no-5p-trim>
 
-DEPRECATED: toggle switch; Disable 5'-trimming (quality and 'N'); May be useful for the identification of duplicates when using trimming of reads.
-Duplicates are usually identified by the 5' mapping position which should thus not be modified by trimming. default=off
-This argument will be removed in favor of --trim-args.
+=item Mandatory parameters
 
-=item B<--disable-zipped-output>
+distmap --reference-fasta /reads/dmel_genome.fasta --output /reads/bwa/bwa_mapping
+--input "/reads/5M_reads_1.fastq,/reads/5M_reads_2.fastq"
+--mapper bwa --mapper-path /usr/local/hadoop/bwa --mapper-args "-l 200"
 
-DEPRECATED: This option is ignored.
+=item If pseudo-mandatory options are not in the PATH
+ 
+distmap --reference-fasta /reads/dmel_genome.fasta --output /reads/bwa/bwa_mapping
+--input "/reads/5M_reads_1.fastq,/reads/5M_reads_2.fastq"
+--mapper bwa --mapper-path /usr/local/hadoop/bwa --mapper-args "-l 200"
+--hadoop-home /usr/local/hadoop
+--readtools /usr/local/share/java/ReadTools.jar
+--picard /usr/local/share/java/picard.jar
 
+=item Hadoop-related job management
+ 
+distmap --reference-fasta /reads/dmel_genome.fasta --output /reads/bwa/bwa_mapping
+--input "/reads/5M_reads_1.fastq,/reads/5M_reads_2.fastq"
+--mapper bwa --mapper-path /usr/local/hadoop/bwa --mapper-args "-l 200"
+--queue-name pg1 --job-priority VERY_HIGH --job-desc "BWA mapping 5M 1"
 
-=back
+=item Trimming when uploading to HDFS
+ 
+distmap --reference-fasta /reads/dmel_genome.fasta --output /reads/bwa/bwa_mapping
+--input "/reads/5M_reads_1.fastq,/reads/5M_reads_2.fastq"
+--mapper bwa --mapper-path /usr/local/hadoop/bwa --mapper-args "-l 200"
+--trim-args "--trimmer MottQualityTrimmer --mottQualityThreshold 20
+--readFilter ReadLengthReadFilter --minReadLength 50 --disable5pTrim"
 
-=head1 Details
+=back
 
- DistMap is an integrated pipeline to map FASTQ formatted short reads on a local Hadoop cluster framework.
- It takes all input from local disk and return single SAM or BAM mapping file in local disk. User does not need to upload or download data in Hadoop manually.
 
 
 =head1 AUTHORS
 
- Ram Vinay Pandey
- Christian Schloetterer
+The original code for distmap was written by Ram Vinay Pandey and
+Christian Schloetterer as described in the citation.
+
+Version 3 is a major improvement of the code developped by Rupert
```

Comment:
one "p" too many

Reply:
Did you fixed this? Sorry that I didn't catch it before merging...

Reply:
PR for the fix -> #74 (merged directly)